testsuite/tests/singlecluster/limitador/route/test_route_rule.py

```diff
@@ -2,23 +2,21 @@
 
 import pytest
 
-from testsuite.gateway import RouteMatch, PathMatch, MatchType
-from testsuite.kuadrant.policy.rate_limit import Limit, RouteSelector
+from testsuite.kuadrant.policy.rate_limit import Limit
+from testsuite.kuadrant.policy.authorization import Pattern
 
 pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
 
 
 @pytest.fixture(scope="module")
 def rate_limit(rate_limit):
     """Add limit to the policy"""
-    selector = RouteSelector(
-        RouteMatch(path=PathMatch(value="/get", type=MatchType.PATH_PREFIX)),
-        RouteMatch(path=PathMatch(value="/anything/test", type=MatchType.PATH_PREFIX)),
-    )
-    rate_limit.add_limit("multiple", [Limit(5, 10)], route_selectors=[selector])
+    when = [Pattern("request.path", "eq", "/get")]
```

Comment:
Missing path for `/anything/test`

Reply:
This wouldn't work due to the comment above, but I think the `test_rule_mismatch` test should be deleted altogether as it doesn't work in the current state and it can be reconsidered with introduction of section names (expected behavior when targeting a non-existent section)